.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord CIL community
+    url: https://discord.com/invite/9NTWu9MEGq
+    about: Contact our developers and fellow users in the Discord CIL community
+  - name: Contact us
+    url: https://ccpi.ac.uk/cil/
+    about: Other ways to get in touch
+  - name: Contributors PR
+    url: https://github.com/TomographicImaging/CIL/compare/master..
+    about: Open a pull request
+  - name: Developers PR
+    url: https://github.com/TomographicImaging/CIL/compare/master..
```

Comment:
is this correct?

Reply:
No, waiting for PR template to be merged to 
add permanent link